src/Aws/Common/Credentials/Credentials.php

```diff
@@ -147,7 +147,7 @@ public static function fromIni($profile = null, $filename = null)
             throw new \RuntimeException("Invalid AWS credentials file: {$filename}.");
```

Comment:
just for readability, I'd probably move the assignment of parsing the ini file into it's own line and then add another conditional to ensure that it is not false.